.editorconfig

```diff
@@ -0,0 +1,13 @@
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+indent_size = 4
+indent_style = space
+insert_final_newline = true
+trim_trailing_whitespace = true
+xcode_trim_whitespace_on_empty_lines = true
```

Comment:
Does Xcode not respect the unadorned version of this option?

Reply:
Xcode has had longstanding behavior that treats trailing whitespace on empty lines distinctly from trailing whitespace on non-empty lines and so the standard editorconfig key applies only to the former. This extra nonstandard key gets us the reasonable behavior we want.

Reply:
Thanks, Jake! I’ve wondered about that for a while now - I appreciate the explanation 😊